src/modules/nfts/components/wizard/PriceSummary.tsx

```diff
@@ -104,9 +105,16 @@ export default function PriceSummary({
                 <Paragraph style={{ fontSize: 14, opacity: 0.6 }}>
                   Estimated network fee x{files.length}
                 </Paragraph>
-                <Paragraph style={{ fontSize: 14 }}>
-                  <span className="sol-icon">◎</span> {SOL_COST_PER_NFT}
-                </Paragraph>
+                <div
+                  style={{
+                    fontSize: 14,
+                    display: 'flex',
+                    alignItems: 'center',
+                    marginTop: '-0.9rem',
+                  }}
+                >
+                  <SolSymbol size={14} /> <span>{SOL_COST_PER_NFT}</span>
```

Comment:
I recommend wrapping the sol symbol in a styled component. Apply these extra styles to it.

Reply:
Yeah good idea, will do that in both cases